tokamak/1-no-currents/BOUT.inp

```diff
@@ -0,0 +1,309 @@
+
+NOUT = 100
+TIMESTEP = 1000
+
+TwistShift = false  # use twist-shift condition?
+ballooning = true
+shiftinitial = false
+
+MZ = 1     # number of points in z direction (2^n + 1)
+
+# NXPE = 10
+
+ZPERIOD=1
+
+grid = tokamak-example.nc
+
+[restart]
+init_missing = true
+
+[mesh]
+paralleltransform = shifted
+
+symmetricGlobalX = true
+
+shiftangle_from_zshift=true
```

Comment:
If I remember right, this was a workaround for hypnotoad2 grids when they did not include `ShiftAngle`. hypnotoad2 does calculate and save `ShiftAngle` now, so I think this can go.